### ISSUE TITLE
Autosize of column-width seems to work invalid

### ISSUE THREAD

**Issue by jjoschyy** [randym/axlsx#125]
Using Excel 2010, auto-column-width seems to work invalid. There is a column-resize, but it is always about 30% too small. 

Furthermore, is there a way to set a default column width?

Thanks a lot,
Joachim

**Comment by dsmalko** [comment]
Having same issue. Is there a way to get auto-width working?

**Comment by Absox** [comment]
Still having this issue.

**Comment by dimarymar** [comment]
Have similar issue. I've set each width to zero and its not very like i want. Seems like it do autosize instead

**Comment by toncid** [comment]
I don't think that Excel supports column auto-width, thus making the formula simply a best guess.

**Comment by jaspertandy** [comment]
@toncid I don't think that's true, because double-clicking a cell divider sets the width to be the correct value so it must be able to do it.

**Comment by toncid** [comment]
@jaspertandy My understanding is that, on double-click, Excel does the column width calculation and saves the width (in pixels or points) in the document. I'm not aware of the "auto" width value that is supported by the XSLX format.
